packages/@glimmer/runtime/lib/syntax/functions.ts

```diff
@@ -275,12 +279,16 @@ export class PartialInvoker implements DynamicInvoker<ProgramSymbolTable> {
       locals[name] = ref;
     }
 
-    let evalScope = outerScope.getEvalScope()!;
+    if (outerEvalScope === null) {
+      outerEvalScope = {};
+    }
+
+    let mergedScope = Object.assign(outerEvalScope, locals);
```

Comment:
We can't rely on `Object.assign` in this version, we have a shared `assign` utility method that we can use though.

https://github.com/glimmerjs/glimmer-vm/blob/1759c16defc546b034b97e37141187652ed93859/packages/%40glimmer/util/lib/object-utils.ts#L10

Reply:
updated, thanks